### DISCUSSION TITLE
How to automatically start nvim-tree on startup?

### DISCUSSION THREAD

**Discussion by archiif** [NvChad/NvChad#879] [Q&A]
This is my `chadrc.lua`:

```lua
local M = {}

M.ui = {
   theme = "onedark",
}

M.plugins = {
   status = {
      alpha = true,
   },
   default_plugin_config_replace = {
      nvim_tree = { 
        open_on_setup = true, 
      },
   },
}

return M
```

But nvim-tree is still not automatically opened on startup. What am I doing wrong here?

**Comment by siduck** [comment]
![image](https://user-images.githubusercontent.com/59060246/158707293-1f2090d4-b684-491f-b57a-40131218da34.png)
set this as false in your chadrc (refer default_config.lua ) and then do packercompile

**Reply by archiif** [reply]
I tried this, but it still doesn't work. I did use `:PackerCompile`, but it doesn't seem to do anything (at least it's not showing any messages).
This is my current chardrc:
```lua
local M = {}

M.ui = {
   theme = "onedark",
}

M.plugins = {
   status = {
      alpha = true,
   },
   options = {
      nvimtree = {
         lazy_load = false,
      },
   },
   default_plugin_config_replace = {
      nvim_tree = {
        open_on_setup = true,
      },
   },
}

return M
```
